db/data/regions/BR/en.yml

```diff
@@ -8,10 +8,12 @@ en:
             default: Street and house number
             optional: Street and house number (optional)
           errors:
-            blank: Enter a street and house number
+            blank_instructional: Enter a street and house number
+            blank_informative: Street and house number is missing
```

Comment:
Grammar: `Street and house number are missing`, or if it's only one or the other, `Street or house number is missing`